docker/pypi/wmagent-couchdb/Dockerfile

```diff
@@ -50,7 +51,7 @@ ADD manage ${COUCH_MANAGE_DIR}/manage
 RUN ln -s ${COUCH_MANAGE_DIR}/manage ${COUCH_ROOT_DIR}/manage
 
 # The $COUCH_CONFIG_DIR is to be mounted from the host and locla.ini read from there
-ADD local.ini ${COUCH_CONFIG_DIR}/local.ini
+ADD local.ini ${COUCH_DEPLOY_DIR}/local.ini
```

Comment:
I feel like this file should remain under COUCH_CONFIG_DIR. But let me see how these tests go.

Reply:
It remains under `$COUCH_CONFIG_DIR`, this one here is just the template to be copied  and properly edited later under `$COUCH_CONFIG_DIR` during the `manage init`  step. And for the rest of the container lifetime it is always searched for and referred from `COUCH_CONFIG_DIR` The problem was: at built time, we cannot have any idea where `COUCH_CONFIG_DIR` would be mounted.